app/controllers/application_controller.rb

```diff
@@ -1,9 +1,9 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  def current_user
-    @current_user = @user || User.first
-  end
+  # def current_user
+  #   @current_user = @user || User.first
+  # end
```

Comment:
- [OPTIONAL] Kindly remove commented code to keep it clean

Reply:
Yeah Sure, Thanks.